### PR TITLE
feat: Change height of empty conversations list while we have filtered groups

### PR DESCRIPTION
**Pull request by przemvs**
## Description

While we have empty conversations list and has results in groups, now we have changed height of the empty conversation component.

## Screenshots/Screencast (for UI changes)

image

## Checklist

- [ ] mentions the JIRA issue in the PR name (Ex. [WPB-423])
- [ ] PR has been self reviewed by the author;
- [ ] Hard-to-understand areas of the code have been commented;
- [ ] If it is a core feature, unit tests have been added;

[WPB-423]: https://wearezeta.atlassian.net/browse/WPB-423?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by e-maad** [comment]
@przemvs 
Some checks are failing.

**Comment by tlebon** [comment]
does it make sense even to show the "no results" component in this scenario?

**Comment by przemvs** [comment]
> does it make sense even to show the "no results" component in this scenario?

https://wearezeta.atlassian.net/browse/WPB-11097?focusedCommentId=128831 - there You have a ticket with details ;)

File: src/script/page/LeftSidebar/panels/Conversations/ConversationView/ConversationView.tsx [hunk 1]
```diff
@@ -41,15 +41,30 @@ export const ConversationView = ({
...
+    return null;
+  }
+
+  return (
+    <ul css={{margin: 0, paddingLeft: 0}} data-uie-name="conversation-view">
+      {conversations.map((conversation, index) => (
+        <ConversationListCell key={conversation.id} {...getCommonConversationCellProps(conversation, index)} />
+      ))}
+    </ul>
+  );
```
Comment (e-maad) [inline src/script/page/LeftSidebar/panels/Conversations/ConversationView/ConversationView.tsx]: Is there any way to remove the redundant code?

File: src/script/page/LeftSidebar/panels/Conversations/ConversationsList.tsx [hunk 2]
```diff
@@ -148,21 +146,14 @@ export const ConversationsList = ({
...
-        </ul>
-      </div>
+      <ConnectionRequests connectionRequests={connectRequests} onConnectionRequestClick={onConnectionRequestClick} />
+
+      <ConversationView
+        conversations={conversations}
+        conversationsFilter={conversationsFilter}
+        currentFolder={currentFolder}
+        getCommonConversationCellProps={getCommonConversationCellProps}
+      />
```
Comment (e-maad) [inline src/script/page/LeftSidebar/panels/Conversations/ConversationsList.tsx]: If we pass conversations (filtered/unfiltered), we can prevent `ConversationView` from redundant code.

```
 const filteredConversations = currentFolder?.conversations().filter(conversationSearchFilter(conversationsFilter));
 const conversationsToDisplay = filteredConversations.length ? filteredConversations : conversation;
 
 // then
 
      <ConversationView
        conversations={conversationsToDisplay}
        getCommonConversationCellProps={getCommonConversationCellProps}
      />
 
```

File: src/script/page/LeftSidebar/panels/Conversations/FilteredGroupConversations/FilteredGroupConversations.tsx [hunk 3]
```diff
@@ -86,14 +86,18 @@ export const FilteredGroupConversations = ({
...
   return (
-    <div>
+    <>
       <h3 css={headingTitle}>{t('searchGroups')}</h3>
 
-      <ul css={css({margin: 0, paddingLeft: 0})} data-uie-name="conversation-view">
+      <ul
+        css={css({margin: 0, paddingLeft: 0})}
+        data-uie-name="filtered-conversation-view"
+        className="filtered-conversations"
```
Comment (e-maad) [inline src/script/page/LeftSidebar/panels/Conversations/FilteredGroupConversations/FilteredGroupConversations.tsx]: As you added a css class only for this `<ul>`, would be nice if we move the css there too.

**Review by e-maad — COMMENTED** [review]
Amazing work!
Added few suggestions 🚀

File: src/script/page/LeftSidebar/panels/Conversations/ConversationView/ConversationView.tsx [hunk 4]
```diff
@@ -19,37 +19,19 @@
...
-}: ConversationViewProps) => {
-  const {currentTab} = useSidebarStore();
-  const isFolderView = currentTab === SidebarTabs.FOLDER;
-
-  if (isFolderView && currentFolder) {
-    return currentFolder
-      ?.conversations()
-      .filter(conversationSearchFilter(conversationsFilter))
-      .map((conversation, index) => (
+export const ConversationView = ({conversations, getCommonConversationCellProps}: ConversationViewProps) => {
```
Comment (e-maad) [inline src/script/page/LeftSidebar/panels/Conversations/ConversationView/ConversationView.tsx]: I think we can remove this component completely.
You moved the logic to `ConversationsList.tsx`

**Review by e-maad — APPROVED** [review]

**Pull request merged**